nextcloud/android
# Feature - Auto Rename

**Demo**

https://github.com/user-attachments/assets/c5178c16-5953-4af7-9e3c-52abf17ea22d


**How to Test?**

1. Enable Windows file compatibility capability
 
2. Attempt to upload files with invalid filenames. You can refer to the `AutoRenameTests` class for guidance.

3. Ensure the filenames are renamed according to the requirements outlined in this [issue](https://github.com/nextcloud/files-clients/issues/62).

4. Try uploading the same files to an encrypted folder.
